ansible/roles/nomad/templates/server_vault.hcl.j2

```diff
@@ -0,0 +1,9 @@
+#
+# // Vault Integration
+#
+
+vault {
+  enabled = true
+  address = "http://{{vault_server}}:8200"
+  #token   = "REPLACETOKEN"
```

Comment:
I know the token is commented out, but why not delete it entirely?

Reply:
in case it doesn't work i can revert